modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectActionLocalServiceTest.java

```diff
@@ -2320,6 +2331,12 @@ private void _assertWebhookObjectAction(
 			status,
 			JSONUtil.getValue(
 				payloadJSONObject, "JSONObject/objectEntry", "Object/status"));
+		Assert.assertEquals(
+			birthday,
+			JSONUtil.getValue(
+				payloadJSONObject,
+				"JSONObject/objectEntryDTO" + objectDefinition.getShortName(),
+				"JSONObject/properties", "Object/birthday"));
```

Comment:
I don´t understand the order that already existed here, so it looks good to me

![image](https://github.com/liferay-objects/liferay-portal/assets/34186632/4f962807-67f9-426e-ba47-d0b143398517)